<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TutorialDialog</class>
 <widget class="QDialog" name="TutorialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Center-out demo</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>Start task</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="viewerUI">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Group Box</string>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>301</width>
        <height>461</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Center Out Parameters</string>
       </attribute>
       <widget class="QDoubleSpinBox" name="MovementDuration">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <double>99999.990000000005239</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Hold_duration_start">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <double>99999.990000000005239</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Hold_duration_end">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <double>99999.990000000005239</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Accuracy">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Hold duration start</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Movement Duration</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>100</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Hold duration end</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Accuracy</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Movement sequence</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Viewer UI</string>
       </attribute>
       <widget class="QScrollArea" name="viewerScrollArea">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>241</width>
          <height>411</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>239</width>
           <height>409</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>OptotrakUI</string>
       </attribute>
       <widget class="QScrollArea" name="optotrakScrollArea">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>151</width>
          <height>161</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>149</width>
           <height>159</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="initializeOptotrakButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>10</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Initialize Optotrak</string>
        </property>
       </widget>
       <widget class="QPushButton" name="initializeRealtimeButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>40</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Intialize Realtime</string>
        </property>
       </widget>
       <widget class="QPushButton" name="setZero">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>70</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Set Zero</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
